packages/node/src/ethereum/api.ethereum.test.ts

```diff
@@ -19,8 +19,9 @@ import {
 } from './block.ethereum';
 
 // Add api key to work
-const HTTP_ENDPOINT = 'https://ethereum.rpc.subquery.network/public';
+const HTTP_ENDPOINT = 'https://eth-pokt.nodies.app';
```

Comment:
I don't think we should be moving away from a subquery RPC. We need to work with the team to figure out what is wrong